obs_maven/repo.py

```diff
@@ -27,20 +27,31 @@
 
 
 class Repo:
-    def __init__(self, uri, project, repository):
-        self.uri = uri
-        self.project = project.replace(":", ":/")
-        self.repository = repository
+    def __init__(self, base_url, project, repository, custom_url=None):
```

Comment:
That's here where you get another bug: `custom_url` might be an empty iterator or an empty array. :wink: Additionally you are _iterating_, building an array, so it is plural: `custom_urls`.